common/test/acceptance/tests/video/test_studio_video_editor.py

```diff
@@ -126,6 +126,27 @@ def test_translations_uploading(self):
         self.assertIn(unicode_text, self.video.captions_text)
         self.assertEqual(self.video.caption_languages.keys(), ['zh', 'uk'])
 
+    def test_save_language_upload_no_transcript(self):
```

Comment:
@muhammad-ammar Actually this test checks `if a transcript language is shown in language menu if no transcript file is uploaded` on video which would be same on LMS side as well.